apps/web/src/components/blocksuite/workspace-header/editor-mode-switch/index.tsx

```diff
@@ -51,8 +55,28 @@ export const EditorModeSwitch = ({
     return () =>
       document.removeEventListener('keydown', keydown, { capture: true });
   }, [setSetting, t]);
+
+  const TooltipContent = () => {
```

Comment:
please move it out side and make it to a new component instead of in the function context